templates/Program/program.html.twig

```diff
@@ -27,6 +27,15 @@
         <span id="name" class="h2 program-name">{{ program.name }}</span>
         <span id="name-translation" class="h2 program-name" style="display: none;" lang=""></span>
 
+        {% if app.user and my_program %}
+          <span>
+            <i id="edit-name-button" class="ms-3 material-icons catro-icon-button align-bottom"
+              data-bs-toggle="tooltip" title="{{ 'programs.editName'|trans({}, 'catroweb') }}">
+              edit
+            </i>
+          </span>
+        {% endif %}
+
         {% if not app.user or not my_program %}
```

Comment:
I think this is the else branch for the new if, right?

Reply:
Yup, should be updated 👍